docs/en/sql-reference/functions/encoding-functions.md

```diff
@@ -967,3 +967,122 @@ Result:
 ```response
 1    2
 ```
+
+## Bech32Encode {#bech32encode}
+
+Encodes a binary data string, along with a human-readable part (HRP), using the [Bech32 or Bech32m](https://en.bitcoin.it/wiki/Bech32) algorithms.
+
+**Syntax**
+
+```sql
+Bech32Encode(hrp, data, witver(optional))
```

Comment:
Usually we use brackets to tell that a parameter is optional, for example [see](https://clickhouse.com/docs/sql-reference/functions/date-time-functions#makedatetime64)

Reply:
Maybe it makes sense to make `hrp` an optional parameter too with `bc` as its default value, WDYT? Also it looks more correct to use `1` as the default value for `witver` (because `Bech32` has a vulnerability and `Bech32m` is more secure).

Reply:
I agree about the default for witver, I will change it to 1. For hrp, I don't think a default makes sense because we don't know whether the user wants a Mainnet or Testnet address, and additionally there are other applications of Bech32, such as for different coins, that use different HRPs (e.g. Litecoin: "ltc"). I think we should leave the Bech32 functions as generic implementations of the algorithm, which is not Bitcoin-specific.